build_debian.sh

```diff
@@ -401,8 +401,9 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y in
     auditd                  \
     linux-perf              \
     resolvconf              \
-	lsof                    \
-	sysstat
+    ethtool                 \
```

Comment:
rules/docker-platform-monitor.mk:$(DOCKER_PLATFORM_MONITOR)_BASE_IMAGE_FILES += cmd_wrapper:/usr/bin/ethtool

Reply:
@prgeor , wonder if there is any conflict with pmon docker.

Reply:
@lguohan have asked Nvidia to remove the ethtool from PMON as they no longer need it
https://github.com/sonic-net/sonic-buildimage/pull/5725/

Reply:
I suppose there could be a conflict with the wrapper. To avoid the coflict, in this PR it should remove the ethtool from pmon and then install it to the host directly.  

Run ethtool could require root privilege after installing it on the host, but I am not 100% sure, author of this PR should double-check.  currently, we don't need root privilege to run the wrapper. 

Btw, in a recent PR https://github.com/sonic-net/sonic-buildimage/pull/15856, ethtool was changed to install from Debian instead of build from source.

Reply:
We dont need ethtool. Will remove it.

Reply:
Deleted ethtool

Reply:
> @prgeor , wonder if there is any conflict with pmon docker.

@prgeor - do you feel this comment is addressed?

Reply:
@prgeor confirmed this comment is resolved.